lib/MusicBrainz/Server/Entity/Util/JSON.pm

```diff
@@ -20,7 +20,7 @@ sub add_linked_entity {
     my ($entity_type, $id, $entity) = @_;
 
     my $entities = ($linked_entities->{$entity_type} //= {});
-    unless (defined $entities->{$id}) {
+    unless (!defined $id || defined $entities->{$id}) {
```

Comment:
(For the record, happens because we don't have the ids when creating the type instances in the schema fixup methods.)

Reply:
Commented in https://github.com/metabrainz/musicbrainz-server/pull/790/commits/ecb46fa25d8ff1d6e7aca86de74f810a8b6eb9b0.